[rospy.client][INFO] 2021-05-02 14:49:58,895: init_node, name[/gazebo_odometry_node], pid[23945]
[xmlrpc][INFO] 2021-05-02 14:49:58,895: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-05-02 14:49:58,896: Started XML-RPC server [http://ubuntu:44811/]
[rospy.impl.masterslave][INFO] 2021-05-02 14:49:58,897: _ready: http://ubuntu:44811/
[rospy.init][INFO] 2021-05-02 14:49:58,897: ROS Slave URI: [http://ubuntu:44811/]
[rospy.registration][INFO] 2021-05-02 14:49:58,898: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2021-05-02 14:49:58,898: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2021-05-02 14:49:58,899: Registering publisher topic [/vesc/odom] type [nav_msgs/Odometry] with master
[rospy.init][INFO] 2021-05-02 14:49:58,998: registered with master
[rospy.rosout][INFO] 2021-05-02 14:49:58,999: initializing /rosout core topic
[rospy.rosout][INFO] 2021-05-02 14:49:59,001: connected to core topic /rosout
[rospy.simtime][INFO] 2021-05-02 14:49:59,002: initializing /clock core topic
[rospy.simtime][INFO] 2021-05-02 14:49:59,004: connected to core topic /clock
[rospy.internal][INFO] 2021-05-02 14:49:59,159: topic[/vesc/odom] adding connection to [/better_odom], count 0
[rospy.internal][INFO] 2021-05-02 14:49:59,272: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2021-05-02 14:49:59,952: topic[/gazebo/link_states] adding connection to [http://ubuntu:33169/], count 0
[rospy.internal][INFO] 2021-05-02 14:49:59,952: topic[/clock] adding connection to [http://ubuntu:33169/], count 0
[rospy.core][INFO] 2021-05-02 14:50:16,055: signal_shutdown [signal-2]
[rospy.internal][INFO] 2021-05-02 14:50:16,059: topic[/vesc/odom] removing connection to /better_odom
[rospy.internal][INFO] 2021-05-02 14:50:16,071: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-05-02 14:50:16,071: topic[/clock] removing connection to http://ubuntu:33169/
[rospy.internal][INFO] 2021-05-02 14:50:16,072: topic[/gazebo/link_states] removing connection to http://ubuntu:33169/
[rospy.impl.masterslave][INFO] 2021-05-02 14:50:16,073: signal-2
[rospy.internal][ERROR] 2021-05-02 14:50:16,073: Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 737, in receive_once
    self.stat_bytes += recv_buff(sock, b, p.buff_size)
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 104, in recv_buff
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][WARNING] 2021-05-02 14:50:16,073: Unknown error initiating TCP/IP socket to ubuntu:49839 (http://ubuntu:33169/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 563, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[rospy.core][INFO] 2021-05-02 14:50:16,693: signal_shutdown [atexit]
